server/style.css

```diff
@@ -0,0 +1,4 @@
+img {
+  height: 200px;
+  width: 200px;
+}
```

Comment:
I'm confused on what the purpose of this file is

Reply:
I am really sorry. I will delete it. It was for testing. I deleted the html, I forgot I still have this one.